src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,157 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import React, { useEffect, useState } from "react";
+import { Action, LoadingIndicator } from "../../presentational";
+import "../HealthPreviewSection/HealthPreviewSection.css"
+import language from "../../../helpers/language";
+import { bloodTypeLabs, singleLabs, manyLabs } from "./LabResultsBloodType.previewdata";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import iconBloodType from "./icon-bloodtype.svg";
+import { TermInformation } from "../../presentational/LabResultWithSparkline/LabResultWithSparkline";
+
+export interface LabResultsBloodTypeProps {
+    previewState?: "BloodTypeLabs" | "SingleLabs" | "ManyLabs" | "NoData"
+    summaryResultsOnly?: true | false
+    maximumResults?: number
+    onClick?: () => void
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+interface BloodTypeLab {
+    Type: string,
+    MostRecentValue: string,
+    MostRecentDate: string,
+    TermInformation: TermInformation,
```

Comment:
this isn't used right?  Seems like we can drop the TermInformation entirely from this